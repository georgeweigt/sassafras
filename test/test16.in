* From STAT 650 ;

data ;
input trt $ y @@ ;
datalines ;
A 34 B 29
A 26 B 17
A 21 B 25
A 22 B 28
A 40 B 18
A 20 B 29
A 33 B 32
A 39 B 31
A 42 B 31
;

proc means n mean clm ;
var y ;
class trt ;
