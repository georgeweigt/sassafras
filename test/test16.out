       TRT     Variable     N       Mean     95% CLM MIN     95% CLM MAX
       A       Y            9     30.778          24.090          37.465
       B       Y            9     26.667          22.370          30.964

